engines/query-sparql-link-traversal-solid/package.json

```diff
@@ -295,7 +295,6 @@
     "@comunica/bus-function-factory": "^4.0.2",
     "@comunica/bus-http-invalidate": "^4.0.2",
     "@comunica/bus-query-operation": "^4.0.2",
-    "@comunica/config-query-sparql": "~4.0.1",
```

Comment:
I think this needs to stay.

Reply:
The depcheck task said it is unneeded, so I removed it. :flushed: Maybe it is marked as unneeded, due to `@comunica/config-query-sparql-link-traversal` (the line below this one) already pulls it in as a dependency?

Reply:
Ah, I think you're right. This was necessary precisely because of the old depcheck task that was bugged.
But the depcheck task has been fixed, so this can now be removed indeed.
And the reason this problem did not arise yet, is because the depcheck task was not being run yet in the CI.